### ISSUE TITLE
roll_min and roll_max can mean different things for different roll_options?

### ISSUE THREAD

**Issue by jeanconn** [sot/sparkles#99]
I think we talked about it, but I'm not sure if it is documented anywhere.  So, roll options are configured so the 0-index roll-option will be the supplied/as-schedule/real roll of the observation being reviewed.  the 0-index roll option looks to have roll_max and roll_min just set to be roll.  That's fine, but I think it is conceptually different from roll_min/roll_max on the other "solved" roll options where each is intended to describe the range that you get about the same catalog, right?

**Comment by taldcroft** [comment]
Good point. This can be a future enhancement to make roll_min/max in that 0th-option case work like you said.
